packages/flutter_tools/gradle/aar_init_script.gradle

```diff
@@ -0,0 +1,96 @@
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.maven.MavenDeployer
+import org.gradle.api.plugins.MavenPlugin
+import org.gradle.api.tasks.Upload
+
+void configureProject(final Project project) {
+    if (!project.hasProperty('android')) {
+        throw new GradleException("Android property not found.")
+    }
+    if (!project.android.hasProperty("libraryVariants")) {
```

Comment:
Does this only work on `com.android.library` projects or will it also work on other library plugins like `com.android.dynamic-feature`?

Reply:
The primary use case is `com.android.library` for module and plugins. What is the use case for `com.android.dynamic-feature`?

Reply:
It's a variant of an Android library/module that can be distributed dynamically by Google Play when your app needs it.

It's likely that if people are using Flutter modules for embedding in existing apps that will want to use dynamic features at some point.

Reply:
I see. cc @xster